### PR TITLE
visuals

### PR DESCRIPTION
**Pull request by dvdfu**

### CODE REVIEW

File: src/objects/frog.lua [hunk 1]
```diff
@@ -62,6 +65,22 @@ function Frog:init(objects, x, y, player)
...
+        quads[i] = love.graphics.newQuad((i - 1) * 32, 0, 32, 32, 32 * 4, 32)
+    end
+    self.particles:setQuads(quads)
+    self.particles:setPosition(x, y - 70)
+    self.particles:setOffset(16, 32)
+    self.particles:setParticleLifetime(10)
+    self.particles:setAreaSpread('ellipse', 128, 24)
+
+    self.timer = Timer.new()
+    self.timer:every(4, function()
```
Comment (hamdanjaveed) [inline src/objects/frog.lua]: would we need to cancel this timer when it goes off screen

File: src/chunkSpawner.lua [hunk 2]
```diff
@@ -17,7 +17,7 @@ function Chunk:init(objects, id, player)
 
     if id == 0 then return end
 
-    if math.random(1, 4) == 1 then
+    if math.random(1, 1) == 1 then
```
Comment (hamdanjaveed) [inline src/chunkSpawner.lua]: might wanna undo this

File: src/objects/frog.lua (hunk 1 above)
Comment (dvdfu) [inline src/objects/frog.lua]: if the body is destroyed and the world is freed of reference to the frog, wouldnt the timer be gc'd with the frog?

File: src/chunkSpawner.lua (hunk 2 above)
Comment (dvdfu) [inline src/chunkSpawner.lua]: oops

**Pull request merged**